lumicks/pylake/kymo.py

```diff
@@ -212,6 +212,20 @@ def _to_spatial(self, data):
     def _shape(self):
         return (self.pixels_per_line,)
```

Comment:
Probably rename or add comment explaining, why `Kymo._shape` does not reflect the actual shape of the Kymo (see explanation in comment for property in `Scan._shape`).

Reply:
Indeed. I'm all for changing the name.